orcid-web/src/main/java/org/orcid/frontend/web/controllers/LoginController.java

```diff
@@ -11,13 +11,17 @@
 import org.orcid.core.constants.OrcidOauth2Constants;
 import org.orcid.core.manager.ClientDetailsEntityCacheManager;
 import org.orcid.core.manager.v3.ProfileEntityManager;
+import org.orcid.core.manager.v3.RecordNameManager;
```

Comment:
@leomendoza123 Since we only need to read data, not write, should use read only manager org.orcid.core.manager.v3.read_only.RecordNameManagerReadOnly

Reply:
@leomendoza123 Please change to "You are signed in as"